homeassistant/components/ohme/button.py

```diff
@@ -0,0 +1,70 @@
+"""Platform for button."""
+
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+
+from ohme import ChargerStatus, OhmeApiClient
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import OhmeConfigEntry
+from .entity import OhmeEntity
+
+PARALLEL_UPDATES = 0
+
+
+@dataclass(frozen=True, kw_only=True)
+class OhmeButtonDescription(ButtonEntityDescription):
+    """Class describing Ohme button entities."""
+
+    is_supported_fn: Callable[[OhmeApiClient], bool] = lambda _: True
```

Comment:
we could think about adding that to a base entitydescription if you're planning to use that in even more platforms